src/WebdriverClassicDriver.php

```diff
@@ -1,21 +1,1390 @@
+        $this->setDesiredCapabilities($this->initCapabilities($desiredCapabilities ?? []));
+        $this->webDriverHost = $webDriverHost ?? 'http://localhost:4444/wd/hub';
+        $this->xpathEscaper = $xpathEscaper ?? new Escaper();
+    }
+
+    // <editor-fold desc="Implementation">
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function start(): void
+    {
+        if ($this->isStarted()) {
+            throw new DriverException('Driver has already been started');
+        }
+
+        try {
+            $this->webDriver = RemoteWebDriver::create($this->webDriverHost, $this->desiredCapabilities);
+            $this->applyTimeouts();
+            $this->initialWindowName = $this->getWindowName();
+        } catch (Throwable $e) {
+            throw new DriverException("Could not start driver: {$e->getMessage()}", 0, $e);
+        }
+    }
+
+    /**
+     * Sets the timeouts to apply to the webdriver session
+     *
+     * @param array $timeouts The session timeout settings: Array of {script, implicit, page} => time in milliseconds
+     * @throws DriverException
+     * @api
+     */
+    public function setTimeouts(array $timeouts): void
+    {
+        $this->timeouts = $timeouts;
+
+        if ($this->isStarted()) {
+            $this->applyTimeouts();
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
     public function isStarted(): bool
     {
-        return false;
+        return $this->webDriver !== null;
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function stop(): void
+    {
+        if (!$this->webDriver) {
+            throw new DriverException('Driver has not been started');
+        }
+
+        try {
+            $this->getWebDriver()->quit();
+            $this->webDriver = null;
+        } catch (Throwable $e) {
+            throw new DriverException('Could not close connection', 0, $e);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function reset(): void
+    {
+        $this->getWebDriver()->manage()->deleteAllCookies();
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function visit($url): void
+    {
+        $this->getWebDriver()->navigate()->to($url);
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function getCurrentUrl(): string
+    {
+        return $this->getWebDriver()->getCurrentURL();
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function reload(): void
+    {
+        $this->getWebDriver()->navigate()->refresh();
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function forward(): void
+    {
+        $this->getWebDriver()->navigate()->forward();
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
```

Comment:
please remove all those phpdoc containing only

```php
    /**
     * {@inheritdoc}
     * @throws DriverException
     */
```

or 

```php
    /**
     * {@inheritdoc}
     */
```

Reply:
I removed `@throws DriverException` in all such cases except for reset(), where throwing is new due to documented undefined behaviour.

Keeping the inheritdoc ones is useful since the IDE renders the resolved documentation nicely:
<img width="440" alt="image" src="https://github.com/minkphp/webdriver-classic-driver/assets/230049/548945b9-8c96-4433-a021-f28557c4c1fe">
what do you think?

Reply:
Inheriting the description should already be the behavior of phpdoc when there is no phpdoc on the element.
When *using* the class, PHPStorm might even maybe do it properly (when opening a file, it shows only what this file contains apparently).
The `{@inheritdoc}` tag is only needed when you want to include the inherited description as part of a longer one in the child class (one that adds additional info)

Reply:
PHPDoc inheritance in PHPStorm works well (for users of the class or interface), however when you open this class directly, it won't show documentation of those methods in the same file.

In other words, without inheritdoc we would miss things like exceptions thrown or arguments typehinting from the interface (but this won't impact the end user).

If you still have a strong opinion on this, I can remove it - presumably not a lot of people would be opening this file directly.

Reply:
I suggest avoiding to duplicate the parent phpdoc. It avoids having to update it when we update it in the parent class.

Contributors working on the class (which will be at most 3 devs probably, including you and me) can navigate to the interface in phpstorm for the case we need to see it (and maybe in the future PHPStorm will add this).